static/js/compose.js

```diff
@@ -439,15 +439,28 @@ function check_unsubscribed_stream_for_send(stream_name, autosubscribe) {
 
 function validate_stream_message_mentions(stream_name) {
     const stream_count = stream_data.get_subscriber_count(stream_name) || 0;
+    const realm_count = people.get_active_human_count() || 0;
     wildcard_mention = util.find_wildcard_mentions(compose_state.message_content());
 
     // check if wildcard_mention has any mention and henceforth execute the warning message.
     if (wildcard_mention !== null && stream_count > exports.all_everyone_warn_threshold) {
         if (user_acknowledged_all_everyone === undefined ||
             user_acknowledged_all_everyone === false) {
             // user has not seen a warning message yet if undefined
-            show_all_everyone_warnings();
 
+            if (!page_params.is_admin && stream_count > 20) {
+                // give error if user isn't admin & mentions > 20 users
+                let err_text;
+                if (wildcard_mention === "all" || wildcard_mention === "everyone") {
```

Comment:
These should actually both be `stream_count`; the number of users in the realm is irrelevant as we have no notices that reach people not on the stream the message was sent to.

Reply:
Oh. I’ll revert.